custom_typings/dom.d.ts

```diff
@@ -0,0 +1,11 @@
+declare interface Window {
+  customElements: CustomElementRegistry;
```

Comment:
you might see about getting these added to typescript's lib files, so a user could do something like in their tsconfig
```
{ "types" : [ "es2015", "dom", "customElements" }
```
cc @DanielRosenwasser